components/blocks/index.tsx

```
@@ -0,0 +1,197 @@
+    style,
+
+    className,
+  }: CardGroupBlockType) => (
+    <Card className={className} data-path={dataPath}>
+      {label && (
+        <CardHeader>
+          <CardTitle style={labelStyle}>{label}</CardTitle>
+        </CardHeader>
+      )}
+      <CardContent className={cn("!py-4 px-8 ", label && "!pt-0")}>
+        <GroupContent style={style} children={children} path={dataPath} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)